### PR TITLE
[API] Disable province code change in PUT endpoint

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A
| --------------- | -----
| Branch?         | 1.8
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | not sure
| Deprecations?   | not sure
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Api/Admin/ManagingCountriesContext.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 use ApiPlatform\Core\Api\IriConverterInterface;
 use Behat\Behat\Context\Context;
+use phpDocumentor\Reflection\Types\Collection;
```
Comment (AdamKasp) [inline src/Sylius/Behat/Context/Api/Admin/ManagingCountriesContext.php:18]: is it needed here?

File: src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Province.xml [hunk 2]
```diff
@@ -29,7 +29,6 @@
            <group>country:create</group>
            <group>country:update</group>
```
Comment (GSadee) [inline src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Province.xml:30]: IMO it should also be removed

**Review by GSadee — COMMENTED** [review]
And you should rebase with 1.8 to fix failing build

**Review by lchrusciel — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, @arti0090! :1st_place_medal:

**Pull request merged**